incubator/harbor/templates/mysql/mysql-ss.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1beta2
+kind: StatefulSet
+metadata:
+  name: "{{ template "harbor.fullname" . }}-mysql"
+  labels:
+{{ include "helm.labels" . | indent 4 }}
+    component: mysql
+spec:
+  replicas: 1
+  serviceName: "{{ template "harbor.fullname" . }}-mysql"
+  selector:
+    matchLabels:
+{{ include "helm.matchLabels" . | indent 6 }}
+      component: mysql
+  template:
+    metadata:
+      labels:
+{{ include "helm.labels" . | indent 8 }}
+        component: mysql
+    spec:
+      containers:
+      - name: mysql
+        image: {{ .Values.mysql.image.repository }}:{{ .Values.mysql.image.tag }}
+        imagePullPolicy: {{ .Values.mysql.image.pullPolicy }}
```

Comment:
Instead of including a StatefulSet for mysql would it make sense to depend on the stable mysql or mariadb charts?

Reply:
I would love to, and I had started to implement that, but I will need to get some updates to Harbor to make this easier, currently it would be a lot of extra spaghetti with initcontainers and other shenanigans.